opennms-provision/opennms-detector-simple/src/main/java/org/opennms/netmgt/provision/detector/snmp/Win32ServiceDetector.java

```diff
@@ -63,15 +65,11 @@ public String getWin32ServiceName() {
     public void setWin32ServiceName(String serviceName) {
         m_win32SvcName = serviceName;
         LOG.debug("setWin32ServiceName: setting service name to {}", serviceName);
-        int snLength = serviceName.length();
-        
-        StringBuffer serviceOidBuf = new StringBuffer(SV_SVC_OPERATING_STATE_OID);
-        serviceOidBuf.append(".").append(Integer.toString(snLength));
-        for (byte thisByte : serviceName.getBytes()) {
```

Comment:
Could this just be fixed by specifying the encoding in ```getBytes()```? It might be getting confused since it's using the platform default encoding here.